CHANGELOG.md

```diff
@@ -0,0 +1,52 @@
+# CHANGELOG
+
+## Next
+
+* Bugfixes
+
+  * Forward exit code from plugins
+  * Discard stderr output during bash completion
+  * Support bash completion when no cluster is attached
+
+## 0.7.1
+
+* Features
+
+  * Build the CLI with 1.12-patch.1 (#1370
+
+* Bugfixes
+
+  * Skip browser-based login providers on user/pass credentials
+
+## 0.7.0
+
+* Breaking changes
```

Comment:
Did we deprecate this in previous versions?

Reply:
These were not deprecated, however these environment variables were either causing bugs or security concerns, while not providing much value.